docs/grcli.1.md

```diff
@@ -48,6 +48,17 @@ Print version and exit.
 
 Print executed commands.
 
+# ENVIRONMENT
+
+#### **DPRC**
+
+Set the DPRC - Datapath Resource Container: This value should match the
+one used by DPDK during the scan of the fslmc bus. It is recommended to
+set this on any NXP QorIQ targets. This serves as the entry point for
+grcli to enable autocompletion of fslmc devices manageable by grout.
+While grcli can configure grout without this environment setting,
+autocompletion of the devargs will not be available.
+
```

Comment:
How does rte_eal_init() manage to detect the devices without this variable set?

Could you do the same in grcli?

Reply:
nope, I cannot, any DPDK application have to be started with DPRC env set.
```
static int
rte_fslmc_scan(void)
{
        int ret;
        char fslmc_dirpath[PATH_MAX];
        DIR *dir;
        struct dirent *entry;
        static int process_once;
        int groupid;
        char *group_name;

        if (process_once) {
                DPAA2_BUS_DEBUG("Fslmc bus already scanned. Not rescanning");
                return 0;
        }
        process_once = 1;

        /* Now we only support single group per process.*/
        group_name = getenv("DPRC");
        if (!group_name) {
                DPAA2_BUS_DEBUG("DPAA2: DPRC not available");
                ret = -EINVAL;
                goto scan_fail;
        }
       ...
```

Reply:
Ok that's very sad :(